baseapp/baseapp_test.go

```diff
@@ -256,7 +256,7 @@ func TestBaseAppOptionSeal(t *testing.T) {
 }
 
 func TestSetMinGasPrices(t *testing.T) {
-	minGasPrices := sdk.DecCoins{sdk.NewInt64DecCoin("stake", 5000)}
```

Comment:
revert this change, this is a test for baseapp which is a part of the sdk and independent of the cosmos-hub, and therefore an imaginary token `stake` is more appropriate that `uatom` here